Makefile

```diff
@@ -0,0 +1,48 @@
+.POSIX:
+
+PKG=	libcperciva
+PROGS=
+TESTS=	tests/buildall tests/buildsingles tests/crc32 tests/getopt tests/heap \
```

Comment:
Is the `tests/` prefix necessary here?  Can't it just be added in the code under `test:` ?

Reply:
How's that going to work with stuff like

        for D in ${PROGS} ${TESTS}; do                  \
                ( cd $${D} && ${MAKE} all ) || exit 2;  \
        done

?  also, we might add some `perftests/` stuff, like in kivaloo.

Reply:
The easy solution there is to split that into two loops, one for ${PROGS} and one for ${TESTS}.

I agree that it makes the code more complicated, but I think it *might* be better to have the mess in the boilerplate which never gets touched rather than in the list of tests which will hopefully be having new entries added?

Your call.

Reply:
I disagree; I think that being explicit about the directories is a good thing.  We've been explicit everywhere else in the build systems of various projects; I don't see that removing a bit of repetition in `tests/` is worth changing that.

Reply:
Ok, I'll merge this as-is.